### PR TITLE
Reduce the output level of mvn on lakefsfs build

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: clients/hadoopfs/src/test/resources/log4j.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Set root logger level to DEBUG and its only appender to A1.
-log4j.rootLogger=DEBUG, A1
+# Set root logger level to INFO and its only appender to A1.
+log4j.rootLogger=INFO, A1
```
Comment (arielshaqed) [inline clients/hadoopfs/src/test/resources/log4j.properties:2]: Not a fan of this change.  It leaves _only_ the boring (and many...) log lines.  But our code actually emits a lot of interesting messages at DEBUG level.

**Review by arielshaqed — CHANGES_REQUESTED** [review]
But I *like* the added lines at DEBUG level.  Why remove them?  I claim that either you ignore them (and then you don't care), or you're looking for something -- and then the extra information does help (it helped me while writing the tests).

File: clients/hadoopfs/src/test/resources/log4j.properties (hunk 1 above)
Comment (nopcoder) [reply clients/hadoopfs/src/test/resources/log4j.properties:2]: Added DEBUG level to io.lakefs

**Review by arielshaqed — APPROVED** [review]
Thanks!

**Pull request merged**